# Identify the period of the game

uint8 NO_HALF = 0
uint8 FIRST_HALF = 1
uint8 SECOND_HALF = 2
uint8 OVERTIME_FIRST_HALF = 3
uint8 OVERTIME_SECOND_HALF = 4
uint8 PENALTY_SHOOTOUTS = 5

uint8 data
